ExomeDepth/make_BEDdetail.py

```diff
@@ -18,47 +19,80 @@ def make_merge_dic(merge_samples):
+            vcf_reader = vcf.Reader(vcf_input_file)
             sampleid = vcf_reader.samples[0]
 
             if 'runid' in vcf_reader.metadata:
                 runid = vcf_reader.metadata['runid'][0]
             else:  # get runid from sample name (legacy version)
                 runid = "_".join(vcf_file.split("/")[-1].split("bam")[1].split("_")[1:-2])
 
-            if sampleid in merge_dic and runid in merge_dic[sampleid]: # Check if sample in specific run is merge sample. If so, exclude
-                excluded_samples_file.write("Sample {sampleid} run {runid} is excluded being merge sample\n".format(sampleid=sampleid, runid=runid))
+            if sampleid in merge_dic and runid in merge_dic[sampleid]:
+                """ Check if sample in specific run is merge sample. If so, exclude """
+                excluded_samples_file.write("Sample {sampleid} run {runid} is excluded being merge sample\n".format(
+                    sampleid=sampleid,
+                    runid=runid)
+                )
                 continue
 
-            if "giab" in sampleid.lower() or "control" in sampleid.lower(): # Remove GIAB and Control samples as these should not be included in the results
-                excluded_samples_file.write("Sample {sampleid} run {runid} is excluded being GIAB or Control sample\n".format(sampleid=sampleid, runid=runid))
+            if "giab" in sampleid.lower() or "control" in sampleid.lower():
+                """ Remove GIAB and Control samples as these should not be included in the results """
+                excluded_samples_file.write("Sample {sampleid} run {runid} is excluded being GIAB or Control sample\n".format(
+                    sampleid=sampleid,
+                    runid=runid)
+                )
                 continue
 
             if sampleid not in vcf_file:
-                excluded_samples_file.write("Sample {sampleid} form run {runid} does not have the same ID in VCF file {vcf_file} and is excluded\n".format(sampleid=sampleid, runid=runid, vcf_file=vcf_file))
+                excluded_samples_file.write(
+                    "Sample {sampleid} form run {runid} does not have the same ID "
+                    " in VCF file {vcf_file} and is excluded\n".format(
+                        sampleid=sampleid,
+                        runid=runid,
+                        vcf_file=vcf_file
+                    )
+                )
                 continue
 
             if len(vcf_reader.samples) > 1:  # multisample VCF
-                excluded_samples_file.write("VCF {vcf_file} from run {runid} is excluded because vcf was a multisample VCF\n".format(vcf_file=vcf_file, runid=runid))
+                excluded_samples_file.write(
+                    "VCF {vcf_file} from run {runid} is excluded because vcf was a multisample VCF\n".format(
+                        vcf_file=vcf_file,
+                        runid=runid
+                    )
+                )
```

Comment:
This fits:
```python
excluded_samples_file.write("VCF {vcf_file} from run {runid} is excluded because vcf was a multisample VCF\n".format(
    vcf_file=vcf_file,
    runid=runid
))
```